moped-editor/src/utils/mapHelpers.js

```diff
@@ -1038,6 +1013,27 @@ export const combineLineGeometries = features => {
   return combinedFeaturesCollection.features[0].geometry;
 };
 
+/**
+ * Test the AGOL response JSON for errors. The API will return a 200
+ * response when a query fails.
+ * an error response looks like this:
+ * {
+ *   error: {
+ *      code: 400,
+ *      message: "",
+ *      details: ["'Invalid field: PROJECT_EXTENT_ID' parameter is invalid"],
+ *    },
+ * };
+ * @param {Object} jsonResponse - The response JSON from AGOL
+ * @return {Object} the response JSON, after logging an error if error
+ **/
+const handleAgolResponse = jsonResponse => {
+  if (jsonResponse?.error) {
+    console.error(`Error fetching geometry: ${JSON.stringify(jsonResponse)}`);
+  }
+  return jsonResponse;
+};
+
```

Comment:
addresses a bug in AGOL error handling that surfaced through Chia's work in #546